db/snapshot/backup_task.hh

```diff
@@ -43,6 +43,7 @@ class backup_task_impl : public tasks::task_manager::task::impl {
     comps_map _sstable_comps;
     size_t _num_sstable_comps = 0;
     comps_vector _non_sstable_files;
+    std::vector<sstables::generation_type> _unlinked_sstables;
```

Comment:
`unlinked` does not represent the content of this list.
```suggestion
    std::vector<sstables::generation_type> _single_hard_linked_sstables;
```

Reply:
As I replied elsewhere, I think a better approach would be to refer to the abstract state of the sstable as deleted from the table rather than its state on the filesystem.
This would also serve sstables on object storage.

Reply:
renamed to _deleted_sstable in v5 ([a8bc344](https://github.com/scylladb/scylladb/commit/a8bc344156a32b48877ac1fa90736f07f9d77a6c))